Assets/LeapMotion/Core/Examples/4. Hand Data Shaders/Standard AlphaFade HandProximity.shader

```diff
@@ -0,0 +1,70 @@
+﻿Shader "Leap Motion/Examples/Standard AlphaFade HandProximity" {
```

Comment:
The convention we have been using so far in all our shaders is to exclude spaces.

Reply:
:sob: